cmd/commands/globusCheckTransfer.go

```diff
@@ -0,0 +1,260 @@
+package cmd
+
+import (
+	"crypto/tls"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/paulscherrerinstitute/scicat/cmd/cliutils"
+	"github.com/paulscherrerinstitute/scicat/datasetIngestor"
+	"github.com/paulscherrerinstitute/scicat/datasetUtils"
+	"github.com/spf13/cobra"
+)
+
+var globusCheckTransfer = &cobra.Command{
+	Use:   "globusCheckTransfer [options] (transfer_task_id transfer_task_id ...)",
```

Comment:
So users need to remember the globus task id? It feels more convenient to just use the PID.

Reply:
The task ID is listed by datasetIngestor when ingesting datasets with the globus transfer option. 

If there's a way to add as a tag the globus task id associated with the dataset in SciCat, that would work as well. The tag could be removed by this command afterwards. 

If we are certain that PID's will never exceed 128 characters, another option would be using the "label" attribute of Globus transfers where I can put in the PID of datasets, then use a filter by label when searching for transfers. https://docs.globus.org/api/transfer/task_submit/
![image](https://github.com/user-attachments/assets/bd5a188b-e6d1-4cf5-a303-51ef1549bcd0)

Reply:
I like the idea of using label to store the PID.

Reply:
I'll do this in a later PR, it requires significant rework that involves the Globus library as well.